src/main/java/arsenic/utils/minecraft/PlayerUtils.java

```diff
@@ -21,26 +20,49 @@
 public class PlayerUtils extends UtilityClass {
 
     private static final Minecraft mc = Minecraft.getMinecraft();
+    private static int onGroundTicks, inAirTicks;
+
+    public static int onGroundTicks() {
+        return onGroundTicks;
+    }
+
+    public static int offGroundTicks() {
+        return inAirTicks;
+    }
+
+    public static void updateTicks(boolean onGround) {
+        if (onGround) {
+            onGroundTicks++;
+            inAirTicks = 0;
```

Comment:
make a separate class for this and register it with event bus and use events to update stuff

Reply:
no need to complicate

Reply:
this way of doing it is really a bad idea

Reply:
make a separate class to keep track of things